packages/service-integration/src/service_integration/osparc_config.py

```diff
@@ -86,12 +86,14 @@ def create_default(
                 }
             }
         )
+        return model
 
     @classmethod
     def from_yaml(cls, path: Path) -> "DockerComposeOverwriteCfg":
         with path.open() as fh:
             data = yaml_safe_load(fh)
-        return cls.parse_obj(data)
+        model: "DockerComposeOverwriteCfg" = cls.parse_obj(data)
```

Comment:
just curious, is there some special reason that they are in double quotes? thanks

Reply:
Its a forward declaration since the class is still not defined (not that is a class method of the same class)